EddystoneBeacon.cpp

```diff
@@ -117,3 +144,35 @@ unsigned char EddystoneBeacon::compressURI(const char* uri, char *compressedUri,
 void EddystoneBeacon::loop() {
   this->poll();
 }
+/* loop() for TLM below:**/
+float temperature;
+int B=3975;                  //B value of the thermistor
+float resistance;
+unsigned long pdu_count = 0;
+void EddystoneBeacon::tempLoop(int analogPin) {
```

Comment:
I don't think this needs be here, the existing ```::loop()``` function can be used.

Also, we should have a ```setTLMData(int temperature, int batteryLevel);``` API. The temperature sensor code can go in an example sketch instead.